instadeepai/Mava
# feat: make best checkpoint support norm params

## What?
Make the bestcheckpoint component support the case of normalization
## Why?
Currently the best checkpointer and the absolute metric features are working fine in the default case, however, they both don't support the case of the normalization of the params such as normalize observation and normalize the target value.
## How?
Edit the stored network in the best_checkpoint params
## Extra
Close #859 
### To test this feature:
1. Run `examples/smac/feedforward/decentralised/calculate_absolute_metric.py` to check the logged json file
2. Run `examples/smac/feedforward/decentralised/best_checkpointed_net.py` to check that it restore the best network
